### DISCUSSION TITLE
Rendering issues with PrimeNG Ultima 17 update

### DISCUSSION THREAD

**Discussion by nexgen-ui** [primefaces/community#792] [PrimeNG Templates] [answered]
There are a couple of issues with the styling in the latest update.

The mega menu text colors are hard-coded to red in the dark theme _variables.scss file.

New update

$menuitemTextColor:red !default;

Should be

$menuitemTextColor:$textColor !default;

Also the table demo rating icons are too big compared to the previous release

![Screenshot 2023-12-30 161215](https://github.com/primefaces/community/assets/6703726/96889bcf-e5b9-4fc9-923f-6d9d3ee39ddc)

![Screenshot 2023-12-30 161303](https://github.com/primefaces/community/assets/6703726/af62b24e-9644-40c7-8e1a-5a3d71310c3e)

The blog demo text and images are not appearing as well

![image](https://github.com/primefaces/community/assets/6703726/2cf615df-cdee-4232-b1c2-9abd11899904)

There might be other issues, these are just the ones that I found so far.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for reporting the issue, it might be a leftover from the testing process. We'll fix and update the v17 from the store. It'll be available in 10-20mins
